### ISSUE TITLE
Only can train several epochs ?

### ISSUE THREAD

**Issue by BonGum** [facebookresearch/SlowFast#658]
Every time I train the model for about ten epochs，procedure finally goes to fail and it shows me that...
![problem](https://github.com/facebookresearch/SlowFast/assets/58102607/75b84f20-2c3e-4caa-9f49-08d3803c7658)
In .yaml，I set the DECODING_BACKEND:pyav，TRAIN.BATCH_SIZE: 2 , NUM_WORKERS:0. It terribly weried! 
my gpu is only one GeFore RTX 3060 Laptop, and my train data is just eight about 10s video clips.
Can anyone help me?

**Comment by BonGum** [comment]
Below is my .yaml file
TRAIN:
  ENABLE: True
  DATASET: mydata
  BATCH_SIZE: 2
  EVAL_PERIOD: 10
  CHECKPOINT_PERIOD: 1
  AUTO_RESUME: False
DATA:
  NUM_FRAMES: 8
  SAMPLING_RATE: 8
  TRAIN_JITTER_SCALES: [256, 320]
  TRAIN_CROP_SIZE: 224
  TEST_CROP_SIZE: 256
  INPUT_CHANNEL_NUM: [3]
  DECODING_BACKEND: pyav
  PATH_TO_DATA_DIR: C:\Users\YXY\Desktop\Video_Understanding\SlowFast-main\data\MyData
RESNET:
  ZERO_INIT_FINAL_BN: True
  WIDTH_PER_GROUP: 64
  NUM_GROUPS: 1
  DEPTH: 50
  TRANS_FUNC: bottleneck_transform
  STRIDE_1X1: False
  NUM_BLOCK_TEMP_KERNEL: [[3], [4], [6], [3]]
NONLOCAL:
  LOCATION: [[[]], [[]], [[]], [[]]]
  GROUP: [[1], [1], [1], [1]]
  INSTANTIATION: softmax
BN:
  USE_PRECISE_STATS: True
  NUM_BATCHES_PRECISE: 200
SOLVER:
  BASE_LR: 0.1
  LR_POLICY: cosine
  MAX_EPOCH: 100
  MOMENTUM: 0.9
  WEIGHT_DECAY: 1e-4
  WARMUP_EPOCHS: 34.0
  WARMUP_START_LR: 0.01
  OPTIMIZING_METHOD: sgd
MODEL:
  NUM_CLASSES: 400
  ARCH: i3d
  MODEL_NAME: ResNet
  LOSS_FUNC: cross_entropy
  DROPOUT_RATE: 0.5
TEST:
  ENABLE: True
  DATASET: kinetics
  BATCH_SIZE: 64
DATA_LOADER:
  NUM_WORKERS: 0
  PIN_MEMORY: False
NUM_GPUS: 1
NUM_SHARDS: 1
RNG_SEED: 0
OUTPUT_DIR: .

**Comment by kienld304** [comment]
you can set MAX_EPOCH: 1 to quicly see checkpoint
